docs/docs/reference/impersonation.md

```diff
@@ -32,12 +38,133 @@ In certain circumstances, it's useful for an administrative user to impersonate
 
 :::
 
-## Example
-
 Here's what it looks like.
 
 <video width="100%" height="600" controls=""><source src="./img/pomerium-user-impersonation.mp4" type="video/mp4">
 Your browser does not support the video tag.
 </video>
 
+### Using the command line interface
+
+Pomerium also includes a command line interface (cli) for generating arbitrary route-scoped service account sessions. Generated service accounts can be used to impersonate users, perform service-to-service communication, and facilitate end-to-end testing for applications managed by Pomerium. The cli is especially useful in situations where an administrator needs more control over the sessions she generates, or if skipping the authentication portion of pomerium's flow is required.
+
+### How
+
+```bash
+pomerium-sa generates a pomerium service account from a shared key.
+
```

Comment:
thought: do you mean `pomerium-cli` here or that's intended?

Reply:
Yes, good catch. Thank you.